airflow/gcp/hooks/gcs.py

```diff
@@ -323,6 +323,26 @@ def delete(self, bucket_name, object_name):
 
         self.log.info('Blob %s deleted.', object_name)
 
+    def delete_bucket(self, bucket_name: str):
+        """
+        Delete an bucket object from the Google Cloud Storage
+
+        :param bucket_name: name of the bucket which will be deleted
+        :type bucket_name: str
+        """
+
+        client = storage.Client()
+        bucket = client.bucket(bucket_name)
+
+        if not bucket.exists():
+            self.log.warning("Bucket {} not exists".format(bucket_name))
+            return
+        self.log.info("Bucket name is: {}".format(bucket_name))
+
+        self.log.info("Deleting {}".format(bucket.name))
+        bucket.delete()
```

Comment:
Does it return True/False? If yes maybe it's worth to return this information? WDYT?

Reply:
Doesn't return anything. In your opinion should be here any info after operation?